src/segments/git.go

```diff
@@ -237,6 +245,25 @@ func (g *Git) Commit() *Commit {
 			g.commit.Subject = line
 		case "ha:":
 			g.commit.Sha = line
+		case "rf:":
+			refs := strings.Split(line, ", ")
+			for _, ref := range refs {
+				ref = strings.TrimSpace(ref)
+				switch {
+				case ref == "HEAD":
+					// skip "HEAD"
```

Comment:
maybe add a continue here rather than empty case

Reply:
pretty new to Go so just checking - won't `continue` unnecessarily check the the next case condition?

Reply:
oh ok forget that, of course I could check myself and I did - it won't

Reply:
Did so, also changed the case's condition to ignore remotes' `HEAD` references as well (not just local `HEAD`)